swarm/pss/pss_test.go

```diff
@@ -606,7 +623,12 @@ func testNetwork(t *testing.T) {
 		}
 	}
 
-	messagedelayhigh := 0
+	// setup workers
+	jobs := make(chan Job, 10)
+	for w := 1; w <= 10; w++ {
+		go worker(w, jobs, rpcs, pubkeys, hextopic)
+	}
+
```

Comment:
Nicely done! This makes tests pass even on my computer.

Reply:
Yes! Otherwise we fire too many go-routines and slower machines have a hard time running the tests. My old Macbook Pro was actually crashing on occasion.